NO-TASK - Reorder facets on events list.

bozeak opened the pull request:
#### Description

This commit contains feature export of views display with reordered filters required for Culture project.

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

kasperg committed:
Core: Bumped version to 7.x-6.2.1

This includes fixing DDB React to the latest release, 2.1.0.

Commit:
4731 by Inlead: Prevent loading unrelated templates.

Commit:
4731 by Inlead: Display 0 price events as Free in Nodelist.

Commit:
4731 by Inlead: Support i18n for 'More link' in Nodelist panes.

Commit:
4731 by Inlead: Added dynamic height for news teaser.

Commit:
4731 by Inlead: Use PHP 5.6 syntax for nodelist filter.

Commit:
4731 by Inlead: Fix display of More link.

Commit:
4731 by Inlead: Add fix style for image.

Commit:
4731 by Inlead: Fix display on hover for teaser.

Commit:
4731 by Inlead: Fixed hover on Nodeblock display.

Commit:
4731 by Inlead: Fix wrong indention.

Commit:
4731 by Inlead: Fix display grid image and nodeblock.

Commit:
4731 by Inlead: Allow attributes to be set.

Commit:
4731 by Inlead: Nodelist Promoted node looks to wrong wrapper.

Commit:
4731 by Inlead: Consider not used functions.

holt83 committed:
648: Ensure exported page manager pages is activated

holt83 committed:
4951: Use current sort in subsearch

holt83 committed:
4951: Use current facets in subsearch

holt83 committed:
4951: Add top banner to 25_50_25 layout

holt83 committed:
4951: Switch to plugins and add ajax placeholder

- Use ctools content type plugins for translate and suggestions
suggestion message. These plugins are places in the new top
banner and so we can avoid the replace hack in frontend.
- Add new AJAX placeholder theme, callback and hook which all
subsearch modules now uses.
- Switch to POST request and send the serialized search request
object for the user search. This is passed to the ajax placeholder
callbacks and makes it easier to perform subsearch.

holt83 committed:
4951: Move setting of facet.genreCategory into sub modules

This also moves some dependencies around such that ting_subsearch
is not dependable on opensearch, so ting_subsearch_suggestions can
now be used without opensearch.

holt83 committed:
4951: Fix ting_field_search active profile support

Adds new ting_subsearch_do_subsearch function and removes old
function for performing "secondary" search.

Also adds some comments.

holt83 committed:
4951: Fix wrong variable name

holt83 committed:
4951: Add profile query and facets to bibliotek.dk search link

Ensure that the bibliotek.dk search result will be the same as the
result from the subsearch. Appends filters from the search request
and CQL query from the active ting_field_search profile to the
user search.

holt83 committed:
4951: Perform 0-hit redirect in hook_ting_search_results_prefix()

This removes the old zero hit redirect plugin and performs the
redirect in a better timed hook instead.

Also switch to use drupal_set_message() to display message about
redirect to the user. The message is not a suggestion but more an
informal message to the user and drupal_set_message() is therefore
more appropiate. We can also get away with the "original-search"
query parameter.

This solution now uses session cookie instead which will bypass
Varnish. But the search should be cached and it will only be for
redirects, so it should not be a big deal.

holt83 committed:
4951: Add new ting_subsearch_suggestion_message theme

Also remove the old ting_subsearch_message theme.

It was easier to implement a more specific suggestion message
theme hook, now that the zero hit redirect message doesn't share
the theme and uses drupal_set_message() instead.

holt83 committed:
4951: Update variable names

holt83 committed:
4951: Remove original-search query from zero hit redirect

holt83 committed:
4951: Send search query parameters in placeholder POST back

holt83 committed:
4951: Update ting_subsearch_ajax_placeholder_callback()

- Check that the unserialized object is instance of
TingSearchResultInterface before calling hook.
- Allow other modules to alter placeholder content.
- Allow subsearch modules to return render array from placeholder
callback.

holt83 committed:
4951: Add secondary msg box process function to theme item list

holt83 committed:
4951: Rename subsearch secondary to subsearch bibdk

holt83 committed:
4951: Use settings passed by Drupal behaviors attach

holt83 committed:
4951: Return early if no autocomplete available

holt83 committed:
4951: Use drupal_html_id() in suggestion message

holt83 committed:
4951: Refactor suggestions plugin

holt83 committed:
4951: Refactor subsearch info hook

OIOIOOOI committed:
EO-NOTASK - Fixed width for bibliotek body.

holt83 committed:
4951: Switch to tabbed admin settings form

Remove hook_ting_subsearch_info() hook.

Each subsearch module implement their own admin settings form as
a local task (tab).
Suggstions subsearch is the most general and adds the default tab.

holt83 committed:
4951: Use Drupal ajax method of attaching content

holt83 committed:
4951: Render bibdk item list in template

holt83 committed:
4951: Add ting subsearch configure permission

holt83 committed:
4951: Change name of ajax placeholder hook

holt83 committed:
4951: Scrutinizer

holt83 committed:
4951: Remove suggestions subsearch result min limit

This settings provides litte value so it's removed in favor of
reducing complexity.

holt83 committed:
4951: Update comment as suggested in CR

holt83 committed:
4951: Add ting_subsearch prefix to plugin file names

Apparently the 'content_types' key has no effect and ctools was
just using the file name as identifier for the content type
plugins.

To ensure they are unique we prefix the file names.

This is also how ding_campaign_plus does it.

holt83 committed:
4951: Loose div and move class to <li> in bibdk message box

holt83 committed:
4951: Get final bibdk query from opensearch_pre_execute()

holt83 committed:
4951: Add hook_module_implements_alter() implementation

holt83 committed:
4951: Remove empty line

Co-authored-by: Kasper Garnæs <kasperg@users.noreply.github.com>

holt83 committed:
4951: Fix inline var declarations

Commit:
4813: Added image preprocess function

holt83 committed:
4917: Ensure popup agree button message can be changed

A hidden option from the "opt-in with categories" model was
preventing this form field from being shown.

holt83 committed:
4917: Update default text on popup agree and disagree buttons

holt83 committed:
4891: Override default webform mail template in ddbasic

This way we can remove the sumissions values token when the
default template is used, but retain the possibility to use it
when using "Custom template".

Also remove old hook implementation that removed submission values
from all submissions mail sent by the webform module.

xendk committed:
4325: Configure cropping for page list image

xendk committed:
4919: Let role change action respect role_delegation

xendk committed:
4919: Fixup code style

xendk committed:
4919: Remove ding_staff_admin_users

It's been superseded by admin/people.

kasperg committed:
Merge pull request #1682 from reload/4886-limit-related-materials

4886: Limit related materials to those available at the local library

bozeak committed:
NO-TASK - Fix entity buttons behavior.

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 10c97b04007300ec47b7753bc7087322272b5fb2
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 9cb8be193c67438fa423636a4a3949b419198c13
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fa9da7299a0b8a7e5ce0711dabc43f1fe45e9ca6
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 34811b40be4e86920dd6278911e4722f030a54da
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: cba0cbf483ba7dacbc84f2b716a6888ffdc999e8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a2905378938429312d1ddcbe46b7a18b3098e704
  * Commits must be in the following format: [issue ID] - [short summary of change]
  * Your subject line is longer than 72 characters

 * Commit: bac09fe5426ebd0aa68ba6bf18475a3acbf6a742
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: e65c4724dfda1a223ddf6f2b80ff314c23f888bd
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 0ebbb39ab1ff7c050d48165dbb8182ec5f0381e8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 63477a895bf37d7a52d623998159d0a39eb52d3e
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1306ef73e23f7f663de59d8520dcbe02d6f027b9
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 886baa6d8ceb95633ff94c37893678ec8ddf6b5d
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: f9721b2ba81e8d0b9393e95f11b60c919372eeae
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: bdbf39cc71e9837208630f74c6d1314fc94c0c4a
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 8f2d5f0b793445dcc9305868f49ff5d920f2a418
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 2b83995aea0bdf001f15a73bd746f855a4d6e0c9
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: ddb2146bc98e56dffc8faf18213dc3b7a9e83295
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: aa21d7130bd825793d044161946474228c19d942
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 24ff37efce7abbf0989fc79c212c5ffbc7cba901
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: c4fc7f81d0d5e3a36a79b0108eec0b87847566e2
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: dd971d25f3d2b8035d85eaf15512a2cb14fe02d1
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 06f26fa6a42187928e24ae6b18dca53f3a24f2f9
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 31ca25d647cfe55b3cd168cdc2e51e1094d7452a
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 5ed50319730643a5cebd3f7140eca08cfc14c94d
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

kasperg committed:
Merge pull request #1369 from vejlebib/3916-event-list-fitler

3916: Refactor event list filter to be independent of filter order

kasperg committed:
Merge pull request #1691 from aakb/feature/4813-image-alt

4813: Added image preprocess function

kasperg committed:
Merge pull request #1641 from ding2/revert-1614-revert-1580-feature/4653-oembed

Reintroduce "4653: Patched oembed - regular expression is too large"

kasperg committed:
Merge pull request #1500 from vejlebib/4400-fix-double-encoding-search-result

4400: Avoid double rawurlencode on "Read more"-link

kasperg committed:
Merge pull request #1510 from easySuite/issue-4241

4241 by Inlead: Created pane widget to display Sections.

kasperg committed:
Merge pull request #1666 from reload/4193-headlines

4193 headlines

kasperg committed:
Merge pull request #1667 from reload/1519-bad-query

1519: Fix ding_campagin_plus_search's query munging

kasperg committed:
Merge pull request #1668 from reload/1519-translatable-errors

1519 translatable errors

kasperg committed:
Merge pull request #1677 from reload/4913-fix-notices

4913 fix notices

kasperg committed:
Merge pull request #1681 from vejlebib/4906-update-notice

4906: Update privacy notice on contact form and webforms

kasperg committed:
Merge pull request #1683 from reload/4854-clean-up-opening-hours

4854: Clean up opening hours for deleted nodes

kasperg committed:
Merge pull request #1604 from reload/4772-user-login-form-behaviors

4772: Fix action links triggering login form for anonymous users

kasperg committed:
Merge pull request #1665 from vejlebib/4762-mobilesearch-null-tags

4762 mobilesearch null tags

kasperg committed:
Merge pull request #1574 from vejlebib/4592-update-globaleredirect

4592 update globaleredirect

kasperg committed:
Merge pull request #1679 from reload/4493-search-help

4493: Make search help translatable

kasperg committed:
Merge pull request #1618 from aakb/feature/4814-loan-list-cache

4814: Fixed loan list cache

kasperg committed:
Merge pull request #1669 from reload/4800-eresource-tags

4800: Fix eresource tags styling

kasperg committed:
Merge pull request #1478 from reload/4429-pincode-length

4429: Allow for variable length pincodes

kasperg committed:
Merge pull request #1672 from reload/4116-related-padding

4116: Remove padding from "on the site"

kasperg committed:
Merge pull request #1555 from b14/b4153v2

4153: Removed messages border

kasperg committed:
Merge pull request #1625 from JrgenGNielsen/feature/bug4783_update_CKEditor

4783: update CKEditor to 4.14.0

kasperg committed:
Merge pull request #1481 from easySuite/issue-4436

4436 by Inlead: Display top ranking.

kasperg committed:
Merge pull request #1692 from vejlebib/4917-cookie-buttons-text

4917: Update cookie popup agree and disagree buttons text

kasperg committed:
4975: Wait up to 5 seconds when checking for object covers.

We see multiple failures where a cover cannot be found even though
the subsequent screenshot shows the cover as expected. However Covers
may be loaded asynchronously and may not yet be ready when the test
is conducted.

Consequently we wait a bit. This is the same approach taken on the
search page.

holt83 committed:
4912: Switch to no cookie tracking in ding_webtrekk

kasperg committed:
Merge pull request #1696 from reload/4975-behat-wait-for-object-cover

4975: Wait up to 5 seconds when checking for object covers.

kasperg committed:
Merge pull request #1680 from vejlebib/648-rm-username-in-title

648: Ensure no usernames in title on pages under /user

kasperg committed:
Merge pull request #1693 from vejlebib/4891-re-add-submissions-values

4891: Reintroduce the possibility of using submissions values token in webforms

kasperg committed:
Merge pull request #1685 from reload/4803-rolltab-a11y

4803 rolltab a11y

kasperg committed:
Merge pull request #1687 from aakb/feature/4397-styled-checkbokses

4397: Change styling for checkboxes and radiobuttons

kasperg committed:
Merge pull request #1673 from reload/4896-cleanup-hover

4896: Remove JS hover class

kasperg committed:
Merge pull request #1678 from reload/4638-single-material-overlay

4638: Fix single material overlay

kasperg committed:
Merge pull request #1694 from reload/4325-page-list-image-crop

Configure cropping for page list image

kasperg committed:
Merge pull request #1674 from aakb/feature/#4866-fix-upload-cover-button

4866: Add css to hide coverupload button on carousel

kasperg committed:
Merge branch 'reload-4919-user-roles'

kasperg committed:
Merge branch 'aakb-feature/4810-body-links-with-underline'

kasperg committed:
Merge pull request #1689 from vejlebib/4915-cookies-same-site-patch

4915: Add patch to set samesite cookie attribute for PHP sessions

kasperg committed:
Merge branch 'vejlebib-subsearch'

kasperg committed:
Merge pull request #1698 from reload/bug/issue-4731

Bug/issue 4731

kasperg committed:
4915: Do not use samesite for tests

These do not use HTTPS so we cannot use “SameSite=None”.

Setting an empty variable is tricky with drush so we have to use
eval instead.

kasperg committed:
Merge pull request #1699 from reload/4915-test-no-samesite

4915: Do not use samesite for tests

holt83 committed:
4559: Update Sender and From headers

rimi-itk committed:
4964: Fixed heading level on /arrangementer

rimi-itk committed:
4964: Fixed heading level on /nyheder

rimi-itk committed:
4964: Fixed heading level on /contact

rimi-itk committed:
4964: Fixed heading level on /biblioteker

xendk committed:
4983: Fix JS error

xendk committed:
4983: Fix checkbox styling on event and news pages

xendk committed:
4803: Don't pause tabroll on pause button focus

Mobile browsers leaves the focus hanging on the button after
clicking (really, desktop browsers ought to too?), which means that
the rolltab will stay paused after pressing play.

xendk committed:
4986: Update composer i CI

xendk committed:
4986: Fix dependencies to be Composer 2 compatible

kasperg committed:
Merge pull request #1703 from reload/4983-bugfixes

4983 bugfixes

kasperg committed:
Core: Translation update

kasperg committed:
Merge pull request #1704 from reload/4803-followup

4803: Don't pause tabroll on pause button focus

xendk committed:
4241: Fix section group tablet view

xendk committed:
4241: Add image spacing in section group mobile view

xendk committed:
4241: Add cropping to section image

xendk committed:
4241: Remove unused code and clean up

Commit:
4908: Added permissions for local administrators and local editors

Commit:
4908: Added permissions for local administrators and local editors

holt83 committed:
4989: Update type hint causing issues

This will cause error everytime a ding_group content is updated
and will prevent a full feature revert all on deployments.

holt83 committed:
4976: Rename format function and update to check input

- Remove type hint to fix error when data is not array
- Rename format_strings function to format_data
- Check input in format_data to support both string and array

Seems instance of OpenSearchTingObject doesn't follow the doc
of TingObjectInterface and can both return array or false where
as the doc for the interface says string is returned.

OIOIOOOI committed:
EASYOPAC-1255 - Unsplash alt/title placement patch.

bozeak committed:
Merge pull request #665 from easySuite/EASYOPAC-1255

EASYOPAC-1255 - Unsplash alt/title placement patch.

holt83 committed:
4985: Provide better default values for subsearch triggers

holt83 committed:
4969: Ensure drupal AJAX is loaded

holt83 committed:
4991: Add subsearch overview page as default tab

Also change suggestions admin settings form to normal tab, so that
it doesn't stand out compared to the other subsearch modules.

holt83 committed:
4283: Don't render empty price button

holt83 committed:
4283: Add fallback text if there is no prices

holt83 committed:
4283: Fix warning when rendering table with no prices

xendk committed:
4983: Create function for adding tabroll

Simple function to call rather than magic combination of
drupal_add_js. Make the delay configurable for rolltabs in
ding_nodelist too.

xendk committed:
4983: Fix node block event JS to fix height

holt83 committed:
4985: Change bib_dk default position to 'after'

holt83 committed:
4976: Document types expected by format_data function

holt83 committed:
4969: Attach behaviors when adding holdings HTML

This also ensures that Drupal will process our 'use-ajax'
reservation buttons.

bozeak committed:
EASYOPAC-1256 - Implementation of Readers Compass.

bozeak committed:
EASYOPAC-1256 - Skip core changes.

holt83 committed:
4969: Remove reservation and periodical HTML IDs

These are not used anywhere and should not be needed.

For periodicals with many issues it can result in a lot of IDs in
the holdings table, and all these are submitted back in AJAX calls
on the page and can in rare cases result in errors on periodical
reservation.

holt83 committed:
4996: Change default URL for material list and follow searches

holt83 committed:
3916: Add event list filter not null check to views filter

The feature export also includes some cleanup:

- Remove body field from news rss feed views export. This was not
done in #1540.
- Some rules config export was refactored. Better keep it as
features exported it  to avoid getting overriden status.

Commit:
Merge pull request #666 from easySuite/EASYOPAC-1256

EASYOPAC-1256 - Implementation of Readers Compass.

Commit:
Merge pull request #662 from easySuite/NO-TASK

NO-TASK - Fix entity buttons behavior.

Commit:
Merge branch 'development' into release-easyopac

Commit:
Release: 7.x-6.1.1+easy

holt83 committed:
3916: Make event list filter required

kasperg committed:
Merge pull request #1717 from vejlebib/4996-list-searches-default-urls

4996: Change default URL for material list and follow searches

kasperg committed:
Merge pull request #1714 from reload/4983-more-bugfixes

4983 more bugfixes

kasperg committed:
Merge pull request #1718 from vejlebib/3916-event-list-filter-2

3916: Add event list filter not null check to views filter

kasperg committed:
Merge pull request #1708 from vejlebib/fix-ding-group-update-error

4989: Remove type hint causing issues

kasperg committed:
Merge pull request #1707 from arniloftsson/master

4908: Giving permission to local administrators and local editors to use smart search form

kasperg committed:
Merge pull request #1706 from reload/4241-fixes

4241 fixes

kasperg committed:
Merge pull request #1712 from vejlebib/4991-subsearch-menu-structure

4991: Add subsearch overview page as default tab

kasperg committed:
Merge pull request #1710 from vejlebib/subsearch-default-values

4985: Provide better default values for subsearch triggers

kasperg committed:
Merge pull request #1702 from rimi-itk/Issue-4964

Issue 4964: Heading levels

kasperg committed:
Merge pull request #1701 from vejlebib/4559-contact-mail-headers

4559: Update mail headers contact from

holt83 committed:
4969: Use #attached when adding drupal.ajax library script

Also noticed that the normal reservation button doesn't use
attached when adding drupal.ajax, so fixed that too.

holt83 committed:
4969: Pass jquery wrapper object to attachBehaviors

xendk committed:
5000: Filter out excluded branches when checking for reservability

xendk committed:
5000: Fix up code style

kasperg committed:
Merge pull request #1709 from vejlebib/4976-fix-related-materials-error

4976: Fix related materials carousel error

kasperg committed:
Merge pull request #1711 from vejlebib/4969-fix-periodical-reservation-issue

4969: Fix periodical reservation issue

kasperg committed:
Merge pull request #1721 from reload/5000-reservation-branches

5000: reservation branches

kasperg committed:
Merge pull request #1705 from reload/4986-composer-2

4986 composer 2

kasperg committed:
Merge pull request #1713 from vejlebib/4283-add-p2b-prices-rework

4283: Remove empty "add one more" place2book price

kasperg committed:
Merge pull request #1716 from reload/translation-update

Core: Translation update

Commit:
EO-NOTASK - Fix wrong heading markup.

bozeak committed:
EO-NOTASK - Display Address 2 line on library overview.

xendk committed:
5005: Clone page handler properly

kasperg committed:
5010: Patch ECK to avoid PDO exceptions after clean install

For some reason ECK fails after a clean install - currently seen on
Platform.sh. The issue is hard to track down but based on the
community patching the module seems to be the way to go for now.

kasperg committed:
5010: Remove reference to removal of ECK with P2

The ECK module is now used by Ting Smart Search.

kasperg committed:
5006: Update Drupal Core to 7.75

This addresses https://www.drupal.org/sa-core-2020-013.

bozeak committed:
BS-37 - Add <b> tag as allowed to ding_wysiwyg text format.

https://inlead.atlassian.net/browse/BS-37

Commit:
Merge pull request #669 from easySuite/BS-37

BS-37 - Add <b> tag as allowed to ding_wysiwyg text format.

kasperg committed:
Merge pull request #1726 from reload/5010-patch-eck-avoid-pdoexception

5010: Patch ECK to avoid PDOException

OIOIOOOI committed:
Merge branch 'ddbcms' into issue-4436

OIOIOOOI committed:
4436 by Inlead: Rename amount heading.

OIOIOOOI committed:
4436 by Inlead: Correct agency parameter.

OIOIOOOI committed:
4436 by Inlead: Lists titles and headers.

OIOIOOOI committed:
4436 by Inlead: Provide an edit link for local articles.

OIOIOOOI committed:
4436 by Inlead: Typo's and comments.

OIOIOOOI committed:
4436 by Inlead: Sniffer hints.

kasperg committed:
Core: Run tests on opensearch.addi.dk

oss-services.dbc.dk has been shut down. This causes our tests to
fail. We switch to the test version of 5.2 as is does not have access
control. This makes it suitable to use for our test environment.

https://github.com/DBCDK/OpenSearch-webservice/wiki/OpenSearch-Web-Service

kasperg committed:
Merge pull request #1732 from reload/opensearch-update

Core: Run tests on opensearch.addi.dk

kasperg committed:
Merge pull request #1727 from reload/5006-drupal-775

5006: Update Drupal Core to 7.75

kasperg committed:
Merge pull request #1697 from vejlebib/mapp-no-cookie-tracking

Switch to no cookie tracking

kasperg committed:
Merge pull request #1725 from reload/5005-panel-cloning

5005: Clone page handler properly

OIOIOOOI committed:
4436 by Inlead: Code review fixes.

OIOIOOOI committed:
4436 by Inlead: Previous commit line typo.

kasperg committed:
Merge pull request #1728 from easySuite/issue-4436

4436 by Inlead: Top ranking UI changes.

Commit:
EO-NOTASK - Remove gronbib_frontend module.

bozeak committed:
Merge pull request #670 from easySuite/gronbib-cleanup

EO-NOTASK - Remove gronbib_frontend module.

Commit:
Merge pull request #667 from easySuite/no-task-ad

EO-NOTASK - Display Address 2 line on library overview.

kasperg committed:
Core: Disable test of sorting search results by publication date

This start failing as search results containing works are sorted by
objects which are not immediately visible from the search result
page. The current implementation of the test checks on years on the
search page and thus fail.

Temporarily disable the test while we figure out whether the current
behavior is intentional.

kasperg committed:
Merge pull request #1734 from reload/disable-sort-test

Core: Disable test of sorting search results by publication date

holt83 committed:
5030: Fix subsearch permissions

xendk committed:
5031: Don't disable field_term_page

It caused the term page to be disabled on second save. Why one would
want to disable it in the first place is unknown.

kasperg committed:
Merge pull request #1735 from vejlebib/5030-fix-subsearch-permission

5030: Fix subsearch permissions

kasperg committed:
Merge pull request #1736 from reload/5031-section-pages

5031: Don't disable field_term_page

bozeak committed:
EASYOPAC-1264 - Update media_youtube module.

OIOIOOOI committed:
Merge pull request #672 from easySuite/EASYOPAC-1264

EASYOPAC-1264 - Update media_youtube module.

bozeak committed:
EASYOPAC-1267 - Fix promoted nodes video behavior.

OIOIOOOI committed:
Merge pull request #673 from easySuite/EASYOPAC-1267

EASYOPAC-1267 - Fix promoted nodes video behavior.

bozeak committed:
EASYOPAC-1269 - Lead text is not displayed on ding_news_frontpage.

bozeak committed:
EASYOPAC-1270 - Make module independent of ding_groups and ding_tabroll.

OIOIOOOI committed:
EASYOPAC-1263 - Invoke entity buttons consequently.

bozeak committed:
Merge pull request #677 from easySuite/EASYOPAC-1263

EASYOPAC-1263 - Invoke entity buttons consequently.

Commit:
EO-NOTASK - Limit editor permissions.

Commit:
Merge pull request #676 from easySuite/EASYOPAC-1270

EASYOPAC-1270 - Make module independent of ding_groups and ding_tabroll.

Commit:
Merge pull request #678 from easySuite/clean-up-permissions

EO-NOTASK - Limit editor permissions.

Commit:
EO-NOTASK - Add localadmin permission

Commit:
Merge pull request #679 from easySuite/clean-up-permissions

EO-NOTASK - Add localadmin permission

bozeak committed:
EASYOPAC-1267 - Update manualcrop contrib module.

bozeak committed:
EASYOPAC-1262 - Add bibliofil_lists to make file.

Commit:
Merge pull request #681 from easySuite/EASYOPAC-1262

EASYOPAC-1262 - Add bibliofil_lists to make file.

bozeak committed:
EASYOPAC-1271 - Re-attach behaviors on availability processing.

bozeak committed:
EASYOPAC-1271 - Add missing reservation response result types.

bozeak committed:
EASYOPAC-1271 - Fix reservation buttons behavior.

bozeak committed:
EASYOPAC-1272 - Implement video in Ding Nodelist widgets.

OIOIOOOI committed:
Merge pull request #683 from easySuite/EASYOPAC-1272

EASYOPAC-1272 - Implement video in Ding Nodelist widgets.

bozeak committed:
EASYOPAC-1271 - Remove availability impact over reserve buttons.

Commit:
Merge tag '7.x-6.2.1' into development

Commit:
Merge pull request #668 from easySuite/Hotfix-ms

Commit:
Merge pull request #680 from easySuite/EASYOPAC-1267-2

EASYOPAC-1267 - Update manualcrop contrib module.

Commit:
Merge pull request #675 from easySuite/EASYOPAC-1269

EASYOPAC-1269 - Lead text is not displayed on ding_news_frontpage.

Commit:
Merge pull request #682 from easySuite/EASYOPAC-1271

Commit:
Merge branch 'development' into release-easyopac

Commit:
Release: 7.x-6.2.1+easy

bozeak committed:
EASYOPAC-1276 - Remove video support for nodelist's tabroll.

Commit:
Merge pull request #685 from easySuite/EASYOPAC-1276

EASYOPAC-1276 - Remove video support for nodelist's tabroll.

bozeak committed:
EASYOPAC-1277 - Remove horizontal accordion support from nodelist.

bozeak committed:
EASYOPAC-1277 - Fix Scrutinizer issues.

bozeak committed:
EASYOPAC-1279 - Update secure permmissions module.

OIOIOOOI committed:
Merge pull request #686 from easySuite/EASYOPAC-1277

EASYOPAC-1277 - Remove horizontal accordion support from nodelist.

bozeak committed:
EASYOPAC-1282 - Extend countries list on library node form.

Commit:
Merge pull request #688 from easySuite/EASYOPAC-1282

EASYOPAC-1282 - Extend countries list on library node form.

Commit:
Hotfix: Subsearch modules included in core modules.

Commit:
Merge branch 'hotfix' into development

OIOIOOOI committed:
Merge pull request #687 from easySuite/EASYOPAC-1279

EASYOPAC-1279 - Update secure permmissions module.

Commit:
Hotfix: Upgrade modules.

bozeak committed:
EASYOPAC-1286 - Reuse core image styles.

Commit:
Hotfix: Search expand button is not working on item page.

Commit:
Merge branch 'hotfix' into development

bozeak committed:
EASYOPAC-1286 - Use core image styles.

bozeak committed:
EASYOPAC-1286 - Use core image styles.

mihaelad committed:
NO-TASK - Fixed node_blocks display for eresource and page CTS.

OIOIOOOI committed:
Merge pull request #693 from easySuite/EASYOPAC-1286

EASYOPAC-1286 - Reuse core image styles.

mihaelad committed:
Merge branch 'development' of github.com:easySuite/ding2 into NO-TASK

GitCop commented:
There were the following issues with your Pull Request

 * Commit: ee97531a4dd58604653852c75819f86608313a14
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 4952010d8452daeaf13f14caf06cb7403d04dcef
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 07b350b8be2aaa2b0b91cea5848769f2526b3b68
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 4cfbcd99e56441f157d7f7f3e99a5a2cd22fd49b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a34472be9713eb665ac6fac1942ac581b857b635
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1fe1e935402f66e1df61d0dc89380ab794827ede
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 0f01b24e16d4de730a974a787f6ea52a2e119e77
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a14102f9c71aebdebde4c5e5365a860a35d24d00
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 76f5e9670e529bf841e479778b49c254b340d9af
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 5261d5c072d28661631cbc494a975118a2e1a961
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 61ee7db4d1d0e39940151af319749ca5ab11f22d
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 676088e98dce45f97000924717dcfbfa1416c0ca
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: c61f3bcd59b5e16f1cccc0fc4ce7e4303af4a092
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6954ebd4ba19e88942169d926959d8b38103448b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7c45aeaf6ba18bcc88206313b07cc52c408bf7ab
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: b2bd4d126d3656f74fb8e2465877c4a429faa055
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 44e0c6f00f8ccd138dab63e90c8766408f2bb1cc
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 8dee3d76bbb90b513d011f90e5505ac08167b28d
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d3abfd3b75df47c7c3970d0c6df22e998bf77e83
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d2806c956c1add345a12928f9f1492284ce893d4
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 04f8938747cb7d8356c995dae9d3ce123a2e0628
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d2de223c56c757963094e19aeb6c67a8f1fc8ad5
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1d2564ed54f28d3c6070c26c892d1b0520d38124
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6f4b4f3ab636a004e6096bf4d1a283d85eb7d1c9
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 40361f1c6b0bcc05e35199aa09f9dec6898ab4ac
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 679cf753263ed81cc126dfc57df434d8880ac64c
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6ca8944e5fcfd7941b2800a88697112c6b0b92a2
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7bb4af01e0bd11682f15f7727758590cba3f58f8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a962361dd03a346a6563760d5ab76a9cd3abb666
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7171c7574fa7e4608accbcc10d6ef41653bbc7e3
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

mihaelad committed:
NO-TASK - Changed the code to be more compact.

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 4952010d8452daeaf13f14caf06cb7403d04dcef
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 07b350b8be2aaa2b0b91cea5848769f2526b3b68
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 4cfbcd99e56441f157d7f7f3e99a5a2cd22fd49b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a34472be9713eb665ac6fac1942ac581b857b635
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1fe1e935402f66e1df61d0dc89380ab794827ede
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 0f01b24e16d4de730a974a787f6ea52a2e119e77
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a14102f9c71aebdebde4c5e5365a860a35d24d00
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 76f5e9670e529bf841e479778b49c254b340d9af
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 5261d5c072d28661631cbc494a975118a2e1a961
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 61ee7db4d1d0e39940151af319749ca5ab11f22d
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 676088e98dce45f97000924717dcfbfa1416c0ca
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: c61f3bcd59b5e16f1cccc0fc4ce7e4303af4a092
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6954ebd4ba19e88942169d926959d8b38103448b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7c45aeaf6ba18bcc88206313b07cc52c408bf7ab
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: b2bd4d126d3656f74fb8e2465877c4a429faa055
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 44e0c6f00f8ccd138dab63e90c8766408f2bb1cc
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 8dee3d76bbb90b513d011f90e5505ac08167b28d
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d3abfd3b75df47c7c3970d0c6df22e998bf77e83
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d2806c956c1add345a12928f9f1492284ce893d4
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 04f8938747cb7d8356c995dae9d3ce123a2e0628
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: d2de223c56c757963094e19aeb6c67a8f1fc8ad5
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1d2564ed54f28d3c6070c26c892d1b0520d38124
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6f4b4f3ab636a004e6096bf4d1a283d85eb7d1c9
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 40361f1c6b0bcc05e35199aa09f9dec6898ab4ac
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 679cf753263ed81cc126dfc57df434d8880ac64c
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 6ca8944e5fcfd7941b2800a88697112c6b0b92a2
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7bb4af01e0bd11682f15f7727758590cba3f58f8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: a962361dd03a346a6563760d5ab76a9cd3abb666
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 7171c7574fa7e4608accbcc10d6ef41653bbc7e3
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: c2748c59144888d67e4f183b21ba3e52f4d82e5d
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com
